src/__tests__/lib/containers/entity_finder/details/DetailsView.test.tsx

```diff
@@ -72,8 +77,8 @@ const versionResult: PaginatedResults<VersionInfo> = {
   results: [
     {
       id: entityHeaders[0].id,
-      versionNumber: 5,
-      versionLabel: 'version 5 label',
+      versionNumber: 3,
+      versionLabel: 'version 3 label',
```

Comment:
The actual mock data has a `versionNumber` of 3. Having this conflict with what MSW was returning was causing issues.